### PR TITLE
Add starter policy rule assets and harness scaffolding

### PR DESCRIPTION
**Pull request by blackboxprogramming**
## Summary
- add enforce, observe, and edge-case policy rule YAML examples with inline tests and notifications
- document the canonical rule output schema, sample Slack payload, and high-signal failure output
- scaffold a Go harness interface plus policy fixtures and test matrix wiring

## Testing
- not run (not applicable)

------
https://chatgpt.com/codex/tasks/task_e_68da0cc4d72c83299074829d9db485e8

### CODE REVIEW

File: harness/runner.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	case "notify":
+		_ = EmitComplianceNotification(dec)
+	}
+	return nil
+}
+
+func EmitComplianceNotification(dec RuleDecision) error {
+	payload := map[string]any{
+		"text":     fmt.Sprintf(":rotating_light: %s — %s", dec.Details["message"], dec.Reason),
+		"metadata": map[string]any{"rule_id": dec.RuleID, "reason": dec.Reason},
```
[the remark this answers is not in the document]
Comment (blackboxprogramming) [reply harness/runner.go:40]: codex can you assist! Thanks!
